### DISCUSSION TITLE
LG V50S UEFI

### DISCUSSION THREAD

**Discussion by dj-ti** [Project-Aloha/mu_aloha_platforms#109] [General]
Hello

It's possible to add UEFI for LG V50S with dualboot for Windows 11 ARM and Android. This file in question is lg-mh2lm5g_8G.img

Best Regards

**Comment by sunflower2333** [comment]
sorry, not available now.

**Reply by dj-ti** [reply]
It,s possible to replace boot_a and boot_b file having ripped UEFI via QFIL from LG G8X to LG V50S ? These are the same phones having different RAM and disk size. Unless there is somewhere a direct file with boot-mh2lm5g.img for LG V50S ?

**Reply by sunflower2333** [reply]
you can download the lastest uefi in github aciton of this repo.

**Comment by JohnChchch** [comment]
is new img ok? i tried the one mh2lm5g-uefi-v240722.img    in my v50s, still cannot dual boot.  It show lge crash/core dump
